azure-core/azure/core/pipeline/transport/aiohttp.py

```diff
@@ -81,7 +77,8 @@ def _build_ssl_config(self, **config):
             return ssl_ctx
         return verify
 
-    def _get_request_data(self, request):
+    @staticmethod
+    def _get_request_data(request):
```

Comment:
See the comment made in the RetryPolicy about changing to static methods.

Reply:
changing this back to non static